src/_help/getting-started/concepts.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Main concepts
```

Comment:
Don't we want to have the same title than in the index, which is "Important Concepts"?